app/js/config/CookieVariables.js

```diff
@@ -6,5 +6,6 @@ module.exports = {
   user_logged_in: "iwengjk",
   referrer: 'ehdieleda',
   first_time_marker: 'yijeusheuw',
-  apiUrl: 'kdjndnvjds'
+  apiUrl: 'kdjndnvjds',
+  jwt : 'jwt-token'
```

Comment:
The values are used as keys to store cookies. I believe the values were intentionally made cryptic so cookie viewers will not guess at first glance what the cookie is about. Aside from the jwt-token.